tests/storage/imageupload.go

```diff
@@ -131,7 +149,7 @@ var _ = SIGDescribe("[Serial]ImageUpload", func() {
 	}
 
 	Context("[rook-ceph] Upload an image and start a VMI with PVC", func() {
-		DescribeTable("[QUARANTINE][test_id:4621] Should succeed", func(resource, targetName string, validateFunc func(string, string), deleteFunc func(string), startVM bool) {
```

Comment:
We should not remove the `QUARANTINE` label in the same PR that tries to stabilize the test. After the fixes have landed we need to verify that they actually make the test less flaky for a period of time in the periodics (which execute all the quarantined tests), see https://github.com/kubevirt/kubevirt/blob/main/docs/quarantine.md#getting-tests-out-of-quarantine for details.

Reply:
Got it. Done, thank you